### DISCUSSION TITLE
Account approved but no benefits available yet

### DISCUSSION THREAD

**Discussion by Well300O** [community/community#63683] [GitHub Education]
### Hi everyone,

Hi there,my GitHub account approved for student pack but no benefits are available in account what should I do and should wait if yes then how long?
![Screenshot_2023-08-14-14-30-59-54_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/community/community/assets/115608290/0dcf2205-ed21-4786-b3e3-6a4c3fc18e5f)

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
